geode-core/src/main/java/org/apache/geode/distributed/internal/streaming/StreamingOperation.java

```diff
@@ -209,12 +221,12 @@ public void process(DistributionMessage msg) {
         }
         if (isLast) {
           super.process(msg, false); // removes from members and cause us to
-                                     // ignore future messages received from that member
+          // ignore future messages received from that member
```

Comment:
Maybe take it easy with the auto format because it's generating a lot of unrelated whitespace changes.

Reply:
I'll try and revert all whitespace changes

Reply:
It's fine for this time, this was more of a suggestion for future diffs.